reference/5.1/Microsoft.PowerShell.Core/About/about_Functions.md

```diff
@@ -373,7 +375,8 @@ function.
 
 > [!IMPORTANT]
 > If your function defines a `Begin`, `Process` or `End` block, all of your code
-> must reside inside one of the blocks.
+> must reside inside those blocks. No code will be recognized outside the blocks
+> if **any** of the blocks are defined.
```

Comment:
Please change the bold to italics for emphasis.  Bold has a markup meaning in PowerShell.

Reply:
Fixed